app/src/util/form-schema/manual-input-hierarchy.json

```diff
@@ -1490,6 +1560,835 @@
+            "id": "geographic-railways-activity",
+            "railways-vehicle-type": "vehicle-type-all"
+          },
+          {
+            "id": "geographic-railways-activity",
+            "railways-vehicle-type": "vehicle-type-passenger-trains"
+          },
+          {
+            "id": "geographic-railways-activity",
+            "railways-vehicle-type": "vehicle-type-freight-trains"
+          }
+        ]
+      },
+      {
+        "id": "movement-driver-railways-methodology",
+        "implemented": false
+      }
+    ],
+    "directMeasure": {
+      "id": "railways-direct-measure-methodology",
+      "extra-fields": [
+        {
+          "id": "railways-fuel-type",
+          "options": [
+            "fuel-type-gasoline",
+            "fuel-type-diesel",
+            "fuel-type-lpg",
+            "fuel-type-cng",
+            "fuel-type-biofuel"
+          ]
+        },
+        {
+          "id": "railways-vehicle-type",
+          "options": [
+            "vehicle-type-all",
+            "vehicle-type-passenger-trains",
+            "vehicle-type-freight-trains",
+            "vehicle-type-high-speed-trains",
+            "vehicle-type-tourist-trains"
+          ],
+          "multiselect": true
+        },
+        {
+          "id": "railways-fuel-source",
+          "type": "text"
+        }
+      ]
+    }
+  },
+  "II.2.2": {
+    "methodologies": [
+      {
+        "id": "electricity-consumption-railways-methodology",
+        "activities": [
+          {
+            "id": "electricity-consumption-railways-activity",
+            "unique-by": [
+              "railways-vehicle-type",
+              "railways-electricity-source"
+            ],
+            "extra-fields": [
+              {
+                "id": "railways-vehicle-type",
+                "options": [
+                  "vehicle-type-all",
+                  "vehicle-type-passenger-trains",
+                  "vehicle-type-freight-trains",
+                  "vehicle-type-high-speed-trains",
+                  "vehicle-type-tourist-trains"
+                ]
+              },
+              {
+                "id": "railways-electricity-source",
+                "type": "text"
+              }
+            ]
+          }
+        ],
+        "suggestedActivities": [
+          {
+            "id": "electricity-consumption-railways-activity",
+            "railways-vehicle-type": "vehicle-type-all"
+          },
+          {
+            "id": "electricity-consumption-railways-activity",
+            "railways-vehicle-type": "vehicle-type-passenger-trains"
+          },
+          {
+            "id": "electricity-consumption-railways-activity",
+            "railways-vehicle-type": "vehicle-type-freight-trains"
+          }
+        ]
+      }
+    ],
+    "directMeasure": {
+      "id": "railways-direct-measure-energy-consumption-methodology",
+      "extra-fields": [
+        {
+          "id": "railways-vehicle-type",
+          "options": [
```

Comment:
missing options:
the option: "vehicle-type-all"
"exclusive": "vehicle-type-all"